src/download.ts

```diff
@@ -72,15 +72,22 @@ export async function getMetadata(skylink: string, customOptions = {}) {
   const opts = { ...defaultDownloadOptions, ...this.customOptions, ...customOptions };
   const url = this.getSkylinkUrl(skylink, opts);
 
-  console.log(url);
-
-  const response = await this.executeRequest({
-    ...opts,
-    method: "head",
-    url,
-  });
-
-  return JSON.parse(response.headers["skynet-file-metadata"]);
+  let skynetFileMetadata = {};
```

Comment:
Minor point: you can save a couple lines by returning `skynetFileMetadata` from within the `try` block. No need to declare it here.

Reply:
Good catch.